litmus-portal/frontend/src/components/Targets/ClusterInfo/index.tsx

```diff
@@ -75,8 +75,8 @@ const ClusterInfo: React.FC<ClusterVarsProps> = ({ location }) => {
                 </div>
               </div>
               <div className={classes.expDiv}>
-                {/* Set isDisabled to false when implemented */}
-                <ButtonOutline handleClick={() => {}} isDisabled>
+                {/*
+                <ButtonOutline handleClick={() => {}} isDisabled={false}>
```

Comment:
Button is already disbaled, you don't need to remove it completely.

Reply:
I dissabled that now, we need this for next release.

Reply:
PTAL again @Saranya-jena